R/report_regressions.R

```diff
@@ -1,19 +1,20 @@
 #' @keywords internal
 .report_regressions <- function(model, effsize = "default", ci = 0.95, standardize = "refit", standardize_robust = FALSE, bootstrap = FALSE, iterations = 500, performance_metrics = "all", p_method = NULL, ci_method = NULL, centrality = "median", dispersion = FALSE, test = c("pd", "rope"), rope_range = "default", rope_ci = 1, bf_prior = NULL, diagnostic = c("ESS", "Rhat"), ...) {
+
   info <- insight::model_info(model)
 
   # Sanity checks -----------------------------------------------------------
   # Multiple CIs
-  if (length(c(ci)) > 1) {
+  if(length(c(ci)) > 1){
     warning(paste0("report does not support multiple `ci` values yet. Using ci = ", ci[1]), ".")
     ci <- ci[1]
   }
 
   # Default effsize
-  if (!is.null(effsize) && effsize == "default") {
-    if (info$is_binomial) {
+  if(!is.null(effsize) && effsize == "default"){
```

Comment:
Why did you remove the white paces before `{`?

Reply:
did not voluntarily, maybe I overwrote by mistake a non-stylered version